model/Concept.php

```diff
@@ -680,6 +684,14 @@ public function removeDuplicatePropertyValues($ret, $duplicates)
         return $ret;
     }
 
+    public function removeNotationFromProperties($ret, $isShowNotationAsPropertySet = true)
```

Comment:
Also, this method is unnecessary if you're not adding the notation properties to `$ret` at all, as suggested above.

Reply:
I will fix this too

Reply:
Done